app/controllers/hyrax/dashboard/collections_controller.rb

```diff
@@ -68,7 +68,7 @@ def new
         form
       end
 
-      def show
+      def show_admin
```

Comment:
I don't understand why this method has to be renamed.  That seems like a breaking API change as we're removing the show route for a collection.  Definitely couldn't go into hyrax 2.1.